tardis/io/schemas/debug.yml

```diff
@@ -0,0 +1,37 @@
+$$target: debug.yml
+type: object
+additionalProperties: false
+properties:
+  logging_state:
+    type: boolean
+    default: False
+    description: Logging Status for the Simulation execution
```

Comment:
"Logging state for the simulation execution"

Reply:
Cool 🎐